doc/source/index/getting_started.rst

```diff
@@ -0,0 +1,55 @@
+Getting Started
+---------------
+To use this package you will require access to a Granta MI server
+that includes MI Restricted Substances Reports 2022 R1
+or later.
+
+Installation
+~~~~~~~~~~~~
+The ``ansys.grantami.bomanalytics`` package currently supports Python 3.7 through 3.10 on Windows and Linux.
+
+Install the latest release from `PyPI <https://pypi.org/project/ansys-grantami-bomanalytics/>`_ with:
+
+.. code::
+
+    pip install ansys-grantami-bomanalytics
+
+Alternatively, install the latest from `ansys-grantami-bomanalytics GitHub <https://github.com/pyansys/grantami-bomanalytics>`_ via:
+
+.. code::
+
+    pip install git:https://github.com/pyansys/grantami-bomanalytics.git
+
+For a local "development" version, install with git and poetry:
+
+.. code::
+
+    git clone https://github.com/pyansys/grantami-bomanalytics
+    cd grantami-bomanalytics
+    poetry install
```

Comment:
I believe as of poetry 1.2 `pip install -e .` works, but we should probably be advising people to use poetry. I think this is fine.

Reply:
If you install using pip, as far as I know you won't get any dependency groups, you'll only get the main dependencies.